features/admin.connections.v1/components/edit/settings/idp-certificates/idp-certificates.tsx

```diff
@@ -397,45 +409,63 @@ export const IdpCertificates: FunctionComponent<IdpCertificatesV2Props> = (props
+                    </Heading>
+                </Grid>
+                { shouldShowCertificatesInfo() && (
+                    <Grid md={ 12 } lg={ 6 }>
+                        <Message
+                            data-componentid={ `${ testId }-info-messege` }
+                            type="info"
+                            content={ t("console:develop.features.authenticationProvider.forms." +
+                                "certificateSection.info") }
+                        />
+                    </Grid>
+                ) }
+                { isJWKSEnabled && isPEMEnabled && (
                     <Grid md={ 12 } lg={ 6 }>
-                        { selectedConfigurationMode === "jwks"
-                            ? jwksInputForm
-                            : (
-                                <React.Fragment>
-                                    { certificateForm }
-                                    <Hint>
-                                        Upload certificate(s) for this Identity Provider. You can include
-                                        multiple certificates in case if there&apos;s any certificate rotations.
-                                        You can upload { supportedMimes() } types of certificates.
-                                    </Hint>
-                                </React.Fragment>
-                            )
-                        }
+                        <Switcher
+                            widths="two"
+                            compact
+                            data-testid={ `${ testId }-switcher` }
+                            selectedValue={ selectedConfigurationMode }
+                            onChange={ onSelectionChange }
+                            options={ [
+                                {
+                                    label: t("authenticationProvider:forms." +
+                                        "certificateSection.certificateEditSwitch.jwks"),
+                                    value: (JWKS as CertificateConfigurationMode)
+                                },
+                                {
+                                    label: t("authenticationProvider:forms." +
+                                        "certificateSection.certificateEditSwitch.pem"),
+                                    value: ("certificates" as CertificateConfigurationMode)
+                                }
+                            ] }
+                        />
                     </Grid>
+                ) }
+                <Grid md={ 12 } lg={ 6 }>
+                    { selectedConfigurationMode === JWKS
+                        ? jwksInputForm
+                        : (
+                            <>
+                                { certificateForm }
+                                <Hint>
+                                    Upload certificate(s) for this Identity Provider. You can include
```

Comment:
Shall we i18n the strings?

Reply:
Proceeding as discussed offline since this is just an indentation change.